packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -306,12 +306,12 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
 
 export default React.memo(styled(Overview)`
   .filter--tags {
-    .ui--Dropdown {
-      padding-left: 0;
+    width: 17.857rem;
```

Comment:
Is there a reason why this doesn't take up the available space? It is always quite tricky with these (the mock-ups do it nicely, ie. these are in the same line as the buttons, assuming it takes up space), however as it is here it is a bit of a "random" (not the right word, but hopefully drives the point across) sizing.